resources/views/notification/base-html.blade.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <style>
+            @section('style')
+                blockquote {
+                    background: #f9f9f9;
+                    border-left: 10px solid #ccc;
+                    margin: 1.5em 10px;
+                    padding: 0.5em 10px;
+                    quotes: "\201C""\201D""\2018""\2019";
+                }
+                blockquote:before {
+                    color:#ccc;
+                    content: open-quote;
+                    font-size: 4em;
+                    line-height: 0.1em;
+                    margin-right: 0.25em;
+                    vertical-align: -0.4em;
+                }
+                blockquote p {
+                    display:inline;
+                }
+            @show
+        </style>
+    </head>
+    <body>
+        @yield('content')
+
+        <p>&ndash; The OpenGov Foundation Team</p>
```

Comment:
Having to include the signature in every child template seems repetitive, and since I think it should generally be the same for all outgoing messages, having it in the base makes sense to me. If we are going to use template inheritance for user overrides or find we need to change the signature in one child template for some reason, we can make it a section, but I think we can punt that for now.